scene/main/canvas_item.cpp

```diff
@@ -613,9 +640,34 @@ void CanvasItem::_get_property_list(List<PropertyInfo> *p_list) const {
 		} else {
 			pi.usage = PROPERTY_USAGE_EDITOR | (has_def_value ? PROPERTY_USAGE_CHECKABLE : PROPERTY_USAGE_NONE); // Do not save if not changed.
 		}
-
+#if defined(TOOLS_ENABLED) && defined(MODULE_REGEX_ENABLED)
+		const String pname = pi.name;
+#endif
 		pi.name = "instance_shader_parameters/" + pi.name;
 		p_list->push_back(pi);
+#ifdef TOOLS_ENABLED
+		if (Engine::get_singleton()->is_editor_hint()) {
+			DocData::PropertyDoc prop_doc;
+			prop_doc.name = pi.name;
+#ifdef MODULE_REGEX_ENABLED
+			const RegEx pattern(R"(/\*\*\s([^*]|[\r\n]|(\*+([^*/]|[\r\n])))*\*+/\s*instance\s+uniform\s+\w+\s+)" + pname + R"((?=[\s:;=]))");
+
+			Ref<RegExMatch> pattern_ref = pattern.search(code);
+			if (pattern_ref != nullptr) {
+				RegExMatch *match = pattern_ref.ptr();
+				const RegEx pattern_tip(R"(\/\*\*([\s\S]*?)\*/)");
+				Ref<RegExMatch> pattern_tip_ref = pattern_tip.search(match->get_string(0));
+				RegExMatch *match_tip = pattern_tip_ref.ptr();
+				const RegEx pattern_stripped(R"(\n\s*\*\s*)");
+				prop_doc.description = pattern_stripped.sub(match_tip->get_string(1), "\n", true);
```

Comment:
```suggestion
				const RegEx pattern_tip(R"(\/\*\*([\s\S]*?)\*/)");
				Ref<RegExMatch> pattern_tip_ref = pattern_tip.search(pattern_ref->get_string(0));
				const RegEx pattern_stripped(R"(\n\s*\*\s*)");
				prop_doc.description = pattern_stripped.sub(pattern_tip_ref->get_string(1), "\n", true);
```
The code is from existing code but it isn't needed